bokehjs/src/lib/models/glyphs/box.ts

```diff
@@ -124,32 +123,27 @@ export abstract class BoxView extends GlyphView {
     const x = this.renderer.xscale.invert(sx)
     const y = this.renderer.yscale.invert(sy)
 
-    const hits = this.index.indices({x0: x, y0: y, x1: x, y1: y})
-
-    const result = hittest.create_empty_hit_test_result()
-    result.indices = hits
-    return result
+    const indices = this.index.indices({x0: x, y0: y, x1: x, y1: y})
+    return new Selection({indices})
   }
 
   protected _hit_span(geometry: SpanGeometry): Selection {
     const {sx, sy} = geometry
 
-    let hits: number[]
```

Comment:
What is the value-add of this renaming? These are indices, yes, but specifically they are hit-test results. This name is both longer and less descriptive.

Reply:
The reason is that `hits` can mean multiple things (potentially and actually practically, e.g. `Selection.from_hits()`), whereas `indices` has one intrinsic meaning. This rename isn't strictly necessary, but helped me to understand the code better for the upcoming work, in which I will introduce opaque types for managing indices, replacing naked arrays. Having opaque types in this context will allow future potential optimizations, like storing indices as bitsets, etc.

Reply:
Is there a demonstrated need for that kind of optimization that justifies the user cost? Lists of indices are comprehendible even by inexperienced users writing CustomJS. But more importantly, moving away from the simple lists of indices would break a *lot* of code. Even for 3.0 there definitely would need to be a discussion issue open to the community before proceeding with that sort of change.

Reply:
As for the specific  change at issue, I still don't agree that it is a better name, but I've decided it's not something I am going to balk over.